README.md

```diff
@@ -1,13 +1,4 @@
 # Budibase File Upload Tool
-## Demo Video
-Note: You can also set accepted file types, which is not shown in the video.
-![Example GIF](./assets/ezgif.com-gif-maker%20(1).gif)
-
-
-Or watch it on YouTube at a higher framerate.
-
-
-[![Demo Youtube Video](https://img.youtube.com/vi/WgVQgUfEvhM/0.jpg)](https://www.youtube.com/watch?v=WgVQgUfEvhM)
 ## About
 This component is used for adding file fields inside of forms. It stores data inside a text value (JSON serialized), so it sould work with any text-based database column. This should work with both the Budibase DB and any other external SQL or noSQL DBs as all the files are fully serialized into text. This is useful if you wish to circumvent the default attachment field of Budibase, which stores data in MiniIO, and you instead want to store data directly inside your database. As files can have a lot of data within them, do note that some of the generated text fields can be quite long.
```

Comment:
_should_ is misspelled in this sentence: "It stores data inside a text value (JSON serialized), so it sould work with any text-based database column"

Reply:
yeah that's from the original repo this came from. don't worry too much about that. instead, look at the code changes. we just need this to function, we don't need it to be pretty just yet.